src/Elcodi/Bundle/AttributeBundle/CompilerPass/MappingCompilerPass.php

```diff
@@ -26,11 +26,7 @@
 class MappingCompilerPass extends AbstractMappingCompilerPass
 {
     /**
-     * You can modify the container here before it is dumped to PHP code.
```

Comment:
Even if it is copied from the other method, this text is the textual representation of what this method is intended for, right?